Tic_Tac_Toe_V2.1/test/ConsoleViewTest.java

```diff
@@ -1,18 +1,24 @@
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
 
 public class ConsoleViewTest {
 
+
     @Test
-    public void parseStringCoordsToInt_givenCoordsAsString_ReturnsCoordsAsInts() {
+    public void printMessage_whenGivenConstantAndCalledFromConstructor_printsCorrectMessage(){
```

Comment:
help me what this test is for? :)

Reply:
i think i was checking that i got an array of integers back (aka coordinates). unless you mean, what is the purpose of having the test there at all?